charts/datadog/values.yaml

```
@@ -1613,6 +1626,45 @@ agents:
       # agents.containers.processAgent.ports -- Allows to specify extra ports (hostPorts for instance) for this container
       ports: []
 
+    otelAgent:
+      # agents.containers.traceAgent.env -- Additional environment variables for the trace-agent container
```

all the comments on the new fields are not correct since they are referencing `traceAgent instead of `otelAgent`

```suggestion
      # agents.containers.otelAgent.env -- Additional environment variables for the trace-agent container
```